launcher/ui/pages/instance/VersionPage.cpp

```diff
@@ -243,6 +243,9 @@ void VersionPage::updateVersionControls()
     bool supportsFabric = minecraftVersion >= Version("1.14");
     ui->actionInstall_Fabric->setEnabled(controlsEnabled && supportsFabric);
 
+    bool supportsQuilt = minecraftVersion >= Version("1.17.1");
```

Comment:
technically Quilt could run on any version that intermediary supports

Reply:
I have asked the Quilt guys. This checks out